config/prod/xschildw-shinyproxy.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: xschildw-shinyproxy
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["80","8080"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "IT"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infra"
+  # The resource owner
+  OwnerEmail: "x.schildwachter@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  # AMIId: "ami-0ca7208f904dc1874"      # AWS linux 2 AMI 2.0.20200108 x86_64 HVM gp2
+  AMIId: "ami-0d576d466a205f6db"      # Ubuntu, based on Khai's comment
```

Comment:
AMI Id is wrong, it should be one from example PR: https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/3/files#diff-bec33f3449342aabd607a16486fd016aR20